reporter.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
```

Comment:
We might not need this. I am trying to see if using grep will suffice. Hang tight :)